src/Docker.PowerShell/Cmdlets/StartContainer.cs

```diff
@@ -33,9 +34,9 @@ public class StartContainer : MultiContainerOperationCmdlet
                     throw new ApplicationFailedException("The container has already started.");
                 }
 
-                if (PassThru.ToBool())
+                if (PassThru)
                 {
-                    WriteObject(await ContainerOperations.GetContainerByIdOrName(id, DkrClient));
+                    WriteObject((await ContainerOperations.GetContainersByIdOrName(id, DkrClient)).Single());
```

Comment:
It's a very unlikely case, since id corresponding to more than one container would cause the call to start a few line above this to fail in docker.  It would have to be an ambiguously named/ID'ed container that was created between the call to start and the call to get (when passthru is specified).  I don't know if we care enough to have a better error message; maybe instead we just remove `.Single()` in order to have it return multiple in the unlikely case that more than one matches?  No, that might cause errors in the user's scripts... Maybe I need to think about this more...